parquet/src/main/java/org/apache/iceberg/parquet/ParquetDictionaryRowGroupFilter.java

```diff
@@ -285,12 +284,12 @@ public Boolean or(Boolean leftResult, Boolean rightResult) {
     }
 
     @Override
-    public <T> Boolean in(BoundReference<T> ref, Literal<T> lit) {
+    public <T> Boolean in(BoundReference<T> ref, LiteralSet<T> literalSet) {
       return ROWS_MIGHT_MATCH;
```

Comment:
I think this should be implemented as well.

Reply:
Yep, I agree. 
I think It is probably better to be implemented in the next PR, which I am going to work on it after this one.
Please let me know your thoughts. thanks.

Reply:
Implementing these in a follow-up sounds good to me.

Reply:
Thanks.